configs/community.dbuild

```diff
@@ -588,9 +588,12 @@ build += {
     extra.run-tests: false
   }
 
+  // disable -Xfatal-warnings because of new unused warnings introduced
+  // by https://github.com/scala/scala/pull/5402
   ${vars.base} {
     name: "pcplod"
     uri:  ${vars.uris.pcplod-uri}
+    extra.commands: ${vars.default-commands} [ "removeScalacOptions -Xfatal-warnings" ]
```

Comment:
interesting, I should change the `scalacOptions` used in test to be definitive and not inherit from the build.